Contest.pas

```diff
@@ -401,7 +406,11 @@ procedure TContest.SendMsg(const AStn: TStation; const AMsg: TStationMessage);
   case AMsg of
     msgCQ: SendText(AStn, 'CQ <my> TEST');
     msgNR: SendText(AStn, '<#>');
-    msgTU: SendText(AStn, 'TU');
+    msgTU:
+      // send station ID after 3 consecutive QSOs
+      if (RunMode <> rmHST) and (QsoCountSinceStationID >= 2)
```

Comment:
Is it okay for QsoCountSinceStationID to be 2 or more?

Reply:
Yes, the comparison here occurs prior to incrementing the counter. The counter is incremented after successful transmission of the `TU <my>`. This was necessary to allow the user to cancel the transmission with the `ESC` Key.
I updated the comments and push an update.